core/async_cancel.rs

```diff
@@ -779,6 +775,7 @@ mod tests {
     assert!(!cancel_handles.iter().any(|c| !c.is_canceled()));
   }
 
+  #[cfg(not(miri))]
```

Comment:
Looks like bad formatting here

Reply:
Sorry, I am not sure what you meant here. What are you seeing?